compiler/one-cmds/onelib/CfgRunner.py

```diff
@@ -46,6 +46,15 @@ def __init__(self, path):
         self.import_drivers = [
             'one-import-bcq', 'one-import-onnx', 'one-import-tf', 'one-import-tflite'
         ]
+        # parse group option
+        GROUP_OPTION_KEY = 'include'
+        if self.cfgparser.has_option('onecc', GROUP_OPTION_KEY):
+            groups = self.cfgparser['onecc'][GROUP_OPTION_KEY].split()
+            for o in groups:
+                if o == 'O' or not o.startswith('O'):
```

Comment:
does this mean, group option only can start with `O` ?

Reply:
Yes. `-O1`, `-ONCHW`, etc. It looks natural to start with `O`.

Reply:
OK